.dockerignore

```diff
@@ -1,5 +1,3 @@
-.git
```

Comment:
needed to have a complete .git directory in the container so we have a real Git repository during `pip install .`.